src/configs/env.ts

```diff
@@ -44,6 +44,10 @@ export type ENABLE_ABOUT = typeof ENABLE_ABOUT;
 export const ENABLE_TEAMS = process.env.REACT_APP_ENABLE_TEAMS === 'true';
 export type ENABLE_TEAMS = typeof ENABLE_TEAMS;
 
+/** Do you want to enable the MDA Point plan features? */
+export const ENABLE_MDA_POINT = process.env.REACT_APP_ENABLE_MDA_POINT === 'true';
```

Comment:
Can we add this to the same.env file?

Reply:
I thought it's added here https://github.com/onaio/reveal-frontend/blob/NTD/add-students-page-and-reducer/.env.sample#L15

Reply:
Oops, my bad.  Thanks.